README.md

```diff
@@ -138,6 +132,12 @@ deployed in minutes.
 
 For detailed step-by-step instructions on how to deploy Nebari, check the [Nebari documentation][docs-deploy].
 
+## Nebari HPC
+
+The HPC version of Nebari is based on [OpenHPC](https://openhpc.community/).
+
+> NOTE: The tool is currently under development. Curious? Check out the [Nebari HPC](https://github.com/Quansight/qhub-hpc) repository.
```

Comment:
This paragraph was wrong and likely a victim of a global replace of "QHub" with "Nebari".

Quoting @dharhas from an offline conversation:

> Currently qhub-hpc exists, nebari-hpc doesn't. It is possible that nebari-hpc will exist in the future, but for now we should explain that many of the tools that form nebari can be installed via qhub-hpc.

Here is my rewrite attempt with the limited knowledge I have:

```suggestion
## Nebari HPC

An HPC version of Nebari is currently not available. There is one under development for Nebaris precursor QHub.
Curious? Check out the [QHub HPC](https://github.com/Quansight/qhub-hpc) repository.
```

Feel free to suggest something else.